apps/test/unit/templates/instructions/TeacherFeedbackTest.jsx

```diff
@@ -111,6 +113,56 @@ describe('TeacherFeedback', () => {
   });
 
   describe('viewed as Teacher', () => {
+    it('displays correct message with checkmark if student has viewed their feedback', () => {
+      const props = {
+        ...TEACHER_FEEDBACK_NO_RUBRIC_PROPS,
+        latestFeedback: [
+          {feedback_provider_id: 5, student_seen_feedback: new Date()}
+        ]
+      };
+      const seenByStudentSpy = sinon.spy(i18n, 'seenByStudent');
+      const wrapper = shallow(<TeacherFeedback {...props} />);
+      expect(wrapper.find('FontAwesome').length === 1).to.equal(true);
+      expect(wrapper.contains('today')).to.equal(true);
+      expect(seenByStudentSpy).to.have.been.calledOnce;
+      i18n.seenByStudent.restore();
+    });
+
+    it('displays nicely formatted date if student viewed teacher feedback', () => {
+      const today = new Date();
+      const props = {
+        ...TEACHER_FEEDBACK_NO_RUBRIC_PROPS,
+        latestFeedback: [
+          {feedback_provider_id: 5, student_seen_feedback: today}
+        ]
+      };
+      const wrapper = shallow(<TeacherFeedback {...props} />);
+      expect(wrapper.contains('today')).to.equal(true);
+      let yesterday = new Date(today);
+      yesterday.setDate(yesterday.getDate() - 1);
+      wrapper.setState({latestFeedback: [{student_seen_feedback: yesterday}]});
+      expect(wrapper.contains('yesterday')).to.equal(true);
+      const longTimeAgo = Date.parse('2011-10-10');
```

Comment:
Suggestion: use `yesterday` to create a new date for longTimeAgo (i.e. `const longTimeAgo = yesterday.getDate() - 100`). Otherwise, the test makes assumptions about the clock on the environment where this runs. Given that this is a unit test, we don't want it to fail because the environment clock is set in the past.